src/redux/reducers/index.js

```diff
@@ -0,0 +1,5 @@
+import { combineReducers } from "redux";
```

Comment:
we probably only need one root reducer file, so this can be `reducers.js` under `src/redux`